### DISCUSSION TITLE
CMake build support

### DISCUSSION THREAD

**Discussion by gatkinso** [openssl/openssl#22709] [General Discussion]
Sorry if this has been asked before, however a quick search of this forum didn't turn anything up.

So I am wondering: has there has been any discussion regarding CMake build support?  Seems to me that this would greatly simplify life for many folks who support cross platform building.

I see an unofficial project for openssl-cmake on Github (I will refrain from linking to it here but it is easily found).  It is quite useful for personal projects, but I would be reluctant to use it for professional production software, esp. anything FIPS related.

Thought?  Opinions?  Edicts?  Again, apologies if this is beating a dead horse.

**Comment by levitte** [comment]
For your information:

https://github.com/openssl/openssl/issues/16812
https://github.com/openssl/openssl/issues/10902
